sympy/stats/tests/test_continuous_rv.py

```diff
@@ -726,6 +726,41 @@ def test_precomputed_cdf():
         compdiff = simplify(compdiff.rewrite(erfc))
         assert compdiff == 0
 
+
+def test_precomputed_characteristic_functions():
+    import mpmath
+
+    def test_cf(dist, test_point, support_lower_limit, support_upper_limit):
+        pdf = density(dist)
+        t = Symbol('t')
+        x = Symbol('x')
+
+        # first function is the hardcoded CF of the distribution
+        cf1 = lambdify([t], characteristic_function(dist)(t), 'mpmath')
+
+        # second function is the Fourier transform of the density function
+        f = lambdify([x,t], pdf(x)*exp(I*x*t), 'mpmath')
+        cf2 = lambda t: mpmath.quad(lambda x: f(x,t), [support_lower_limit, support_upper_limit], maxdegree=10)
+
+        # compare the two functions at the given point
+        n1 = cf1(test_point)
+        n2 = cf2(test_point)
+
+        assert abs(re(n1) - re(n2)) < 1e-12
+        assert abs(im(n1) - im(n2)) < 1e-12
+
+    test_cf(Beta('b', 1, 2), 0.5, 0, 1)
```

Comment:
These work but I noticed you are choosing test_point values to fall between `support_lower_limit` and `support_upper_limit`. This wasn't needed since the values of t are in the "frequency" space; the choice of t is not tied to where the distribution is supported. You could have a fixed set of t-values for testing all distributions.